scene/main/node.cpp

```diff
@@ -1442,7 +1442,19 @@ Node *Node::get_node_or_null(const NodePath &p_path) const {
 
 Node *Node::get_node(const NodePath &p_path) const {
 	Node *node = get_node_or_null(p_path);
-	ERR_FAIL_COND_V_MSG(!node, nullptr, "Node not found: " + p_path + ".");
+
+	if (p_path.is_absolute()) {
+		ERR_FAIL_COND_V_MSG(
+				!node,
+				nullptr,
+				vformat(R"(Node not found: "%s" (absolute path attempted from "%s").)", p_path, get_path()));
```

Comment:
Is there much point giving this `attempted from "%s"` info? An absolute path is an absolute path, it doesn't matter where it originates from.

Reply:
I prefer giving as much context as possible from error messages. While absolute paths behave the same from anywhere, I think the context from the absolute path error message should still remain the same as the relative path error message. Otherwise, this could end up being confusing (an error message being less informative depending of the argument you give to a function).